server/express_app.js

```diff
@@ -36,7 +36,6 @@ if (config.get('debug')) {
 }
 
 app.use('/v' + version.getApiVersion(), router);
-app.use('/', (new express.Router()).get('/', router.generateIndexPage));
```

Comment:
I added this: https://github.com/ripple/ripple-rest/pull/297/files#diff-79279ffabc15c88f1859a5273ec21eccR42